src/Color/Scale.purs

```diff
@@ -21,18 +21,29 @@ module Color.Scale
   , blueToRed
   , yellowToRed
   , cssColorStops
+
+  -- more advanced staff
```

Comment:
Can we choose a more useful comment here? Also, do we have to export all of these functions?